plugins/adapters/modbus/srcCxx/ModbusAdapterConfiguration.cxx

```diff
@@ -743,6 +777,15 @@ void ModbusAdapterConfiguration::parse_json_config_string(
                 }
 
                 mace.modbus_register_count_ = static_cast<int>(number_node->u.integer);
+            } else if (element_name == "modbus_slave_device_id") {
+                json_value *number_node = node_object->u.object.values[j].value;
+                if (number_node->type != json_integer) {
+                    throw std::runtime_error(
+                            "Error in the JSON configuration "
+                            "<modbus_slave_device_id>.");
```

Comment:
Isn't this JSON? It looks like XML

Reply:
I've used <> as a way of showing the name and values of different elements (instead of quotes or similar things). This is done in the whole adapter